pulsar-io/canal/src/main/java/org/apache/pulsar/io/canal/CanalSourceConfig.java

```diff
@@ -45,44 +45,44 @@
         required = true,
         defaultValue = "",
         sensitive = true,
-        help = "Username to connect to mysql database")
+        description = "Username to connect to mysql database")
```

Comment:
If we remove `help` from the annotation, whoever has connectors outside this repo will get compile errors. We shouldn't do it unless there's a very strong reason to do so